src/utilities/deprecated.jl

```diff
@@ -24,12 +24,12 @@ Base.@deprecate sum_largest sumlargest
 Base.@deprecate sum_smallest sumsmallest
 
 # broadcasting
-if VERSION<v"0.6.0"
-  .*(x::Value, y::AbstractExpr) = broadcast(*, x, y)
-  .*(x::AbstractExpr, y::Value) = broadcast(*, x, y)
-  .*(x::AbstractExpr, y::AbstractExpr) = broadcast(*, x, y)
-  ./(x::Value, y::AbstractExpr) = broadcast(/, x, y)
-  ./(x::AbstractExpr, y::Value) = broadcast(/, x, y)
-  ./(x::AbstractExpr, y::AbstractExpr) = broadcast(/, x, y)
-  .^(x::AbstractExpr, y::Value) = broadcast(^, x, y)
-end
+#if VERSION<v"0.6.0"
+#  .*(x::Value, y::AbstractExpr) = broadcast(*, x, y)
+#  .*(x::AbstractExpr, y::Value) = broadcast(*, x, y)
+#  .*(x::AbstractExpr, y::AbstractExpr) = broadcast(*, x, y)
+#  ./(x::Value, y::AbstractExpr) = broadcast(/, x, y)
+#  ./(x::AbstractExpr, y::Value) = broadcast(/, x, y)
+#  ./(x::AbstractExpr, y::AbstractExpr) = broadcast(/, x, y)
+#  .^(x::AbstractExpr, y::Value) = broadcast(^, x, y)
+#end
```

Comment:
This block can just be deleted.

Reply:
`deprecated.jl` is not used currently (I just commented out the corresponding line in `Convex.jl`) so we might to remove the whole file?

Reply:
Ok, `deprecated.jl` has been removed